lightning/src/routing/onion_message.rs

```diff
@@ -0,0 +1,283 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! [Onion message] pathfinding lives here. Finding paths for onion messages is necessary for
```

Comment:
nit: If you put the first sentence in its own paragraph, it will be the only one shown in  the 'Modules' overview of the [`lightning::routing` docs](https://docs.rs/lightning/latest/lightning/routing/index.html).
```suggestion
//! [Onion message] pathfinding lives here.
//!
//! Finding paths for onion messages is necessary for
```